feature/gribi/ate_tests/base_hierarchical_route_installation_test/base_hierarchical_route_installation_test.go

```diff
@@ -316,12 +316,12 @@ func testRecursiveIPv4Entry(t *testing.T, args *testArgs) {
 			// for devices that return the nexthop without resolving it recursively. For a->b->c the device returns b
 			// note that b->c is validated in the next code block
 			if got != ateIndirectNH {
+				if nh.GetInterfaceRef().GetInterface() == "" {
+					t.Errorf("next-hop interface-ref/interface not found")
+				}
 				t.Errorf("next-hop is incorrect: got %v, want %v or %v ", got, ateIndirectNH, atePort2.IPv4)
 			}
 		}
-		if nh.GetInterfaceRef().GetInterface() == "" {
```

Comment:
It only checks if the interface is empty, but not if the ref interface is correct. I also don't think it needed here for now. 

I think we can just remove this code, and let https://github.com/openconfig/featureprofiles/pull/1132 handles it later.

Reply:
Sure, will remove this.